app/_locales/en/messages.json

```diff
@@ -1487,6 +1490,9 @@
   "learmMoreAboutGas": {
     "message": "Want to $1 about gas?"
   },
+  "learnAbout": {
+    "message": "Learn about"
```

Comment:
For these types of strings, where they feature a link for part of them, we set the string to:

```Learn about $1```

The for the JSX, we do something like:

```
t('textKey1', [
        <a
          href=""
          className="network-status__tooltip-label"
        >
          {t('textKey2')}
        </a> ])
```

Reply:
updated

Reply:
สวัสดีค่ะ เฮ้อ_เหนื่อย
หนูกำลังติดต่อ_ครูสอนทางฝั่งไทยอยู่นะ

แชร์Code ให้ (ความลับ)ตรงไกนได้บ้าง

>

Reply:
Yes, about the gas cost. I want matic go to polagon chain การ more than that I sent wrong 555

Reply:
https://photos.app.goo.gl/Hn1X9jmT42YUxvNJ7